pulp_rpm/app/serializers.py

```diff
@@ -21,6 +21,14 @@ class PackageSerializer(SingleArtifactContentSerializer):
     keeping fields from the parent class as well. Provide help_text.
     """
 
+    relative_path = serializers.CharField(
+        help_text=_("File name of the package"),
+        required=False,
+        allow_null=True,
+        allow_blank=True,
```

Comment:
This seems a bit odd to allow blank and null but it's required for backwards compatibility. See [the related test](https://github.com/pulp/pulp_rpm/blob/master/pulp_rpm/tests/functional/api/test_crud_content_unit.py#L51-L54).